### PR TITLE
[DEV-323] Allow the ignore list to take both user names and IDs

### PR DESCRIPTION
**Pull request by tristaneuan**

### CODE REVIEW

**Comment by esigler** [comment]
:+1: :ship:

**Comment by tristaneuan** [comment]
@esigler One sec actually, about to push another change that allows mention_names to be ignored as well.

**Comment by tristaneuan** [comment]
@esigler This is ready for another CR when you get a moment

File: README.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ config.handlers.jira.site     = 'https://your.jira.instance.example.com/'
 * `context` (string) - If your instance is in a /subdirectory, put that here. Default: `''`
 * `format` (string) - You can select a compact one line issue summary by setting this parameter to `one-line`. Default: `verbose`
 * `ambient` (boolean) - When set to `true`, Lita will show JIRA issue details when a JIRA issue key is mentioned in chat, outside the context of a command. Default: `false`
-* `ignore` (array) - Prevent ambient JIRA issue detection in certain users' messages. Default: `[]`
+* `ignore` (array) - Prevent ambient JIRA issue detection in certain users' messages. Accepts both user names and IDs. Default: `[]`
```
Comment (esigler) [inline README.md]: `Accepts user names, mention names, and IDs` ?

**Comment by esigler** [comment]
Other than the doc comment, LGTM!

**Pull request merged**